hawc/apps/assessment/urls.py

```diff
@@ -8,6 +9,8 @@
 router.register(r"assessment", api.Assessment, basename="assessment")
 router.register(r"dashboard", api.AdminDashboardViewset, basename="admin_dashboard")
 router.register(r"dataset", api.DatasetViewset, basename="dataset")
+router.register(r"global/blogs", api.BlogViewset, basename="blogs")
```

Comment:
i think the blog route will be deleted, and we can change the log route to `logs`

Reply:
we are also going to be adding the option for jobs not to be associated with an assessment, so would it be cleaner to keep the `global` namespace for these kind of endpoints? or would you rather the url paths just be `logs` and `jobs`?

Reply:
I prefer logs and blogs, since the `dashboard` route is the same kind of thing... but no really strong preference from me if you want to namespace them

Reply:
Fair enough! I'll use `logs` and `jobs`